src/masterless.py

```diff
@@ -25,7 +25,7 @@ def requires_masterless(fn):
     @wraps(fn)
     def wrapper(stackname=None, *args, **kwargs):
         ctx = context_handler.load_context(stackname)
-        ensure(stackname and ctx['project']['aws']['ec2']['masterless'], "this command requires a masterless instance.")
+        ensure(stackname and ctx['aws']['ec2']['masterless'], "this command requires a masterless instance.")
```

Comment:
shouldn't this be `ctx['ec2']['masterless']`?

Reply:
hm, missed this one. you might be right.

Reply:
yes, it happens here: https://github.com/elifesciences/builder/blob/3886100f47ecf87c0c39019c1b001cb345bb92c6/src/buildercore/cfngen.py#L265

I have it all checked out, I'll fix it up